doc/build-osx.md

```diff
@@ -1,116 +1,319 @@
-# macOS Build Instructions and Notes
+# macOS Build Guide
+
+**Updated for MacOS [11.2](https://www.apple.com/macos/big-sur/)**
+
+This guide describes how to build bitcoind, command-line utilities, and GUI on macOS
+
+**Note:** The following is for Intel Macs only!
+
+## Dependencies
+
+The following dependencies are **required**:
+
+Library                                                    | Purpose    | Description
+-----------------------------------------------------------|------------|----------------------
+[Xcode Tools](https://developer.apple.com/xcode/features/) | Build      | macOS build tools
```

Comment:
What are the Xcode Tools? Having this here is going to make users think they need to install Xcode. Which unless it's changed recently, is not required.

Reply:
removed in c180c911b88b2bd2baf2c9c2b24e276787ffb69b